pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -919,6 +919,29 @@ var _ = Describe("Converter", func() {
 			})
 		})
 
+		Context("CPU mpx feature", func() {
+			It("should be nil for s390x", func() {
+				v1.SetObjectDefaults_VirtualMachineInstance(vmi)
+				c.Architecture = "s390x"
+				vmi.Spec.Domain.CPU = &v1.CPU{}
+				domain := vmiToDomain(vmi, c)
+				Expect(domain.Spec.CPU.Features).To(BeNil())
+			})
+
+			It("should be present for amd64", func() {
+				v1.SetObjectDefaults_VirtualMachineInstance(vmi)
+				c.Architecture = "amd64"
+				vmi.Spec.Domain.CPU = &v1.CPU{}
+				mpxCPUFeature := api.CPUFeature{
+					Name:   "mpx",
+					Policy: "disable",
+				}
+				domain := vmiToDomain(vmi, c)
+				Expect(domain.Spec.CPU.Features).To(HaveLen(1))
+				Expect(domain.Spec.CPU.Features[0]).To(Equal(mpxCPUFeature))
```

Comment:
Nit: You can simplify to `Expect(domain.Spec.CPU.Features).To(HaveExactElements(mpxCPUFeature))`

Reply:
done. I have simplified it to use the HaveExactElements.